ARK Server Manager/OpenRCON.xaml

```diff
@@ -22,10 +22,10 @@
         <Label Grid.Row="0" Grid.Column="0" Content="Server IP:"/>
         <TextBox Grid.Row="0" Grid.Column="1" Text="{Binding ServerIP}" TabIndex="0"/>
         
-        <Label Grid.Row="1" Grid.Column="0" Content="Server Port:"/>
-        <TextBox Grid.Row="1" Grid.Column="1" Text="{Binding ServerPort}" TabIndex="1"/>
+        <Label Grid.Row="1" Grid.Column="0" Content="RCON Port:"/>
```

Comment:
Looks like we missed some Localization strings.  Do you want to change these to be bound to strings in the localization file?